coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 98.6
+    "covered_percent": 68.54
```

Comment:
That's a huge drop.  Did you not run the entire test suite?

Reply:
no